templates/rhsm-katello-reconfigure.erb

```diff
@@ -106,13 +106,19 @@ fi
   service goferd status >/dev/null && \
   service goferd restart >/dev/null 2&>1
 
+[ -f /etc/init.d/goferd ] && \
```

Comment:
If goferd is a native systemd unit then this is breaking the restarting. It should be checking for a .service file I think.

Reply:
I was copying [a few lines above this](https://git.io/vxUp2), should that change as well? If so, what do you recommend?